app/models/distribution.rb

```diff
@@ -3,7 +3,10 @@
 
 class Distribution < BaseModel
   belongs_to :distributable, polymorphic: true, touch: true
+  belongs_to :audio_version_template
```

Comment:
Yeah, I think this is a pretty good assumption in our data universe.  Will also help to more explicitly define the purpose of an AudioVersion.